admin/roles-config.js

```diff
@@ -1,8 +1,10 @@
 module.exports = {
   ROLE_SERVICE_MANAGER: 'SERVICE-MANAGER',
   ROLE_TEST_DEVELOPER: 'TEST-DEVELOPER',
+  ROLE_HELPDESK: 'HELPDESK',
```

Comment:
Not sure what this object is doing.  It's a mess - some sort of combination of roles and home pages? 😱

Reply:
this is one thing i forgot to bring up - there should be just one canonical set of declared roles in the admin app

Reply:
@GuyHarwood @jon-shipley already made a note for the follow-up about this.